src/components/List/List.jsx

```diff
@@ -64,6 +70,55 @@ const List = () => {
 
   const clearFilter = () => setInputValue('');
 
+  const groups = (list) => {
+    const listByGroups = [[], [], [], []];
+
+    const today = new Date();
+
+    list.forEach((element) => {
+      const isInactive = element.data().lastDate
```

Comment:
what do you think if you do a destructuring like this:
`const { lastDate, lastEstimate } = element.data() `

Reply:
Thanks @FauricioValencia , the change has been done.